contracts/contracts/coordination/GlobalAllowList.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity ^0.8.0;
+import "@openzeppelin/contracts/access/AccessControlDefaultAdminRules.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "./IRitualAuthorizer.sol";
+import "./Coordinator.sol";
+
+contract GlobalAllowList is AccessControlDefaultAdminRules, IRitualAuthorizer {
```

Comment:
After re-reading this PR, I think a more descriptive name for the interface is `IEncryptionAuthorizer`, and subsequently the contract could be `GlobalEncryptionAllowList` (although not as necessary if the interface is renamed).

Reply:
✔️